app/assets/javascripts/app_layout.js

```diff
@@ -11,7 +11,6 @@ var AppLayout = (function() {
   }
 
   return {
-    initialize: initialize,
```

Comment:
I don't know what this is for but it was just proccing and error?  I removed it and nothing seemed to change.